modulemd/include/modulemd-2.0/modulemd-module-index-merger.h

```diff
@@ -103,6 +103,18 @@ G_BEGIN_DECLS
  *   containing the empty string will be interpreted as removing the
  *   translation. Subsequent processing of a higher modified value may restore
  *   it.
+ *
+ * Merging #ModulemdObsoletes entries with identical module, stream, context
+ * and modified values behaves as follows (when accessed through module):
+ *
+ * - Any obsoletes object that is provided by a single #ModulemdModuleIndex
+ *   will be the obsoletes object in the resulting merged #ModulemdModuleIndex.
+ * - If the #ModulemdModuleIndex inputs have different priorities (not common),
+ *   then the obsoletes from the highest priority #ModulemdModuleIndex will be
+ *   used and the others entirely discarded. (Priority is intended for providing
+ *   a total override, including an on-disk configuration).
+ * - If the repos have the same priority (such as "fedora" and "updates" in the
+ *   Fedora Project) the behaviour is undefined.
```

Comment:
Please explain that the `modified` value on the `modulemd-obsoletes` document will be checked for entries in the same priority level.

Reply:
I think this is already covered in the `Merging #ModulemdObsoletes entries with identical module, stream, context and modified values behaves as follows (when accessed through module)` because it isn't checked otherwise.

Or do you mean when associating it with a stream? Because I think that is a different thing and I would rather describe that `modulemd_module_stream_get_obsoletes_resolved` returns the latest active obsoletes valid for this stream (which is missing in the docs and I really should add it).